packages/node_modules/@ciscospark/widget-roster/src/actions.js

```diff
@@ -0,0 +1,9 @@
+export const UPDATE_WIDGET_STATE = `widget-roster/UPDATE_WIDGET_STATE`;
+export function updateWidgetState(state) {
```

Comment:
We actually aren't using this currently, but we will as soon as we start adding/removing users, so I included it in this PR.